util/util.go

```diff
@@ -2,10 +2,15 @@ package util
 
 import (
 	"fmt"
+	"time"
 
 	ma "github.com/multiformats/go-multiaddr"
 )
 
+const (
+	AvgBlockTime = time.Second * 30
```

Comment:
In testnet is 45s. But in mainet will be 30s as mentioned in the spec.
Leaving 30s here since updating more often doesn't hurt; worst thing that can happen is doing nothing.